### DISCUSSION TITLE
Make Telegram channel

### DISCUSSION THREAD

**Discussion by max5354** [maxrave-dev/SimpMusic#43] [General]
It will be great if you make one Telegram channel so we can discuss about ideas and features request and bugs immediately instead of doing in github

**Comment by maxrave-dev** [comment]
Thanks for the idea. I will create soon

**Reply by max5354** [reply]
How to download "debug version"?

**Comment by maxrave-dev** [comment]
Follow SimpMusic Discussion on Telegram: https://t.me/simpmusic_discuss
